stack.yaml

```diff
@@ -1,30 +1,30 @@
-resolver: lts-14.27
+resolver: lts-16.11
 
 packages:
 - '.'
 
 extra-deps:
 - git: https://github.com/dapphub/dapptools.git
-  commit: af84e2ee0a0654fdaa91186384233cf1731ee7ce
+  # TODO: use an hevm release
+  commit: 3efe39b9faaeb500116c285cce6368670a1a7589
   subdirs:
   - src/hevm
 
-- brick-0.46
 - ghci-pretty-0.0.2
-- multiset-0.3.4.1
+- HSH-2.1.3
+- monoidal-containers-0.6.0.1
 - restless-git-0.7
-- rosezipper-0.2
 - s-cargot-0.1.4.0
 - text-format-0.3.2
+- these-1.0.1
 - tree-view-0.5
-- HSH-2.1.3
-- ipprint-0.6
-- sr-extra-1.46.3.2
-- Unixutils-1.54.1
 - witherable-0.3.5
 - witherable-class-0
 - github: dmjio/semver-range
   commit: d8d9db892ddb6ae267c9bcbc4f6602668433f12a
+- github: leventerkok/sbv # 8.7.5
+  # TODO: sbv 8.8
```

Comment:
what happened here?

Reply:
hevm currently pins 8.7.5 (unofficial release) so we should do the same until they pin 8.8 and release, whereby we can also update and release

Reply:
@ggrieco-tob we can use 8.8 as hevm relaxed this dependency since then

Reply:
Done! Keep in mind that in practice won't change anything in Echidna, since symbolic execution is not used.